packages/faas-functions/update-questionnaire-statuses-faas/src/update-questionnaire-statuses.test.js

```diff
@@ -0,0 +1,18 @@
+jest.mock('./update-questionnaire-statuses');
+const updateQuestionnaireStatuses = require('./update-questionnaire-statuses');
+
+describe('update-questionnaire-statuses', () => {
+  let mockEvent;
+  let mockHttpStatus;
+
+  beforeEach(() => {
+    mockEvent = { body: {} };
+    mockHttpStatus = { httpStatus: 200 };
+  });
+
+  it('should call updateQuestionnaireStatuses', async () => {
+    updateQuestionnaireStatuses.mockImplementation();
+    await updateQuestionnaireStatuses(mockEvent, mockHttpStatus);
```

Comment:
updateQuestionnaireStatuses expects only one parameter, but two params are being passed here. Is that expected?

Reply:
Edited.